Bottom: 3be1797233d09a6b64e99edbe88fb775951bd8a6
Top:    fc7ac8b840823dfbd84d7e3054a5d6dc43907790
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-19 01:37:54 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 99fdec1..be9760e 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2507,7 +2507,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
